README.md

```diff
@@ -2,71 +2,21 @@
 
 This repository contains all of the code that runs on [worldcubeassociation.org](https://www.worldcubeassociation.org/).
 
-## Setup
+----
 
-- Clone this repo! (And navigate into it)
-  ```
-  git clone https://github.com/thewca/worldcubeassociation.org
-  cd worldcubeassociation.org
-  ```
+## Common Queries
+- [Overview of the WCA's software ecosystem](https://docs.worldcubeassociation.org/)
+- [Running the website locally](https://docs.worldcubeassociation.org/contributing/quickstart)
+- [Contributing Guide](https://docs.worldcubeassociation.org/contributing/detailed_contributing_guide.html)
+- [API to read data from the website](https://wca-rest-api.robiningelbrecht.be/)
```

Comment:
This API is not officially endorsed by WST and should not appear on our front page readme

Reply:
What would it take for the API to be endorsed by us? And what are the concerns with it not being endorsed?

I'm strongly in favour of linking to it for pragmatic reasons - it is our most common query, and enables hobbyist devs to quickly find what they're looking for.

Reply:
We could change the wording to be more clear that it is unofficial?

Reply:
The main concern is that people could think we are responsible for fixing errors, shielding feature requests or providing help when your app doesn't work